ormolu.cabal

```diff
@@ -130,7 +130,7 @@ executable ormolu
         th-env >=0.1.1 && <0.2,
         optparse-applicative >=0.14 && <0.18,
         ormolu,
-        text >=0.2 && <3.0
+        text >=2.0 && <3.0
```

Comment:
Why bump the lower bound? Stackage still doesn't have `text-2.x`, so this prevents building Ormolu (easily) with Stack: https://github.com/tweag/ormolu/pull/984

Reply:
It is due to the `textToStringBuffer` function that makes use of the UTF-8 encoding, see e.g. here: https://github.com/tweag/ormolu/pull/963#discussion_r1063453308

W.r.t. stack: I thought that as one already has to touch `extra-deps` due to `ghc-lib-parser-9.4` in the current LTS, this wouldn't be a big deal, but maybe I am missing a different downside (I mostly don't use stack).

Reply:
:+1: as long as there's a reason. The problem is that when specifying a boot library in `extra-deps`, Stack ignores all other boot libraries, so you have to explicitly specify all the boot libraries you need. Plus, bumping text requires bumping other deps like `hashable`, so there's a cascading effect. Not a big deal (personally, I don't think there's much reason to install ormolu/fourmolu as a library anyway)